arduino/hardware_wallet/hardware_wallet.ino

```diff
@@ -33,6 +41,11 @@ void show(String msg, bool done=true){
     }
 }
 
+void send_command(String command, String payload) {
+    String combined = command + String(",") + payload;
+    Serial.println(combined);
```

Comment:
The ","-delimiting is hacky

Perhaps we don't want to use Serial.println. The frontend gets some messages that seem to be just new-line characters and this could be why ...

Reply:
It's a good idea to pass command code with the response. ","-delimiting is ok for now.

It makes sense to switch to raw binary packets later with a form like <magic><len><command_code><payload>
I would like to keep human-readable format also, for debugging, but binary is usually more efficient and easier to work with.